Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
+import ReactDOM from'react-dom';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import ImageThumbnail from './components/HeaderComponents/ImageThumbnail';
+import CardContainer from './components/CardComponents/CardContainer';
+import CardThumbnail from './components/CardComponents/CardBanner';
+
 
 const App = () => {
   return (
     <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+      <div>
+          <HeaderContainer/>
+      </div>
+      <span>
+          <CardContainer/>
+          
+      </span>
     </div>
-  );
-};
-
+  )};
+ReactDOM.render(<CardThumbnail/>, document.getElementById('root'));
```

Comment:
Don't need this at all, only need this in the `index.js`. You only need this to put your whole App to the DOM